### PR TITLE
adding v10 specific information for integration testing

### PR DESCRIPTION
**Pull request by Mantus667**
Adding v10 specific information on how to setup a project, state the bug in the pre 10.3.1 version of the test package and giving a sample on how to setup appsettings.Tests.json and GlobalSetup

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for the PR @Mantus667 ! 💪 

I see there are some warnings about long sentences in both articles.. Would you be up for taking a shot at resolving those? 🤞 
Then I'll run it by our developers for a check mean while 😄

**Comment by Mantus667** [comment]
Hi @sofietoft ,
will have a look if I can shorten/restructure the sentences ab bit. :)

**Comment by Zeegaan** [comment]
Hi @Mantus667, did you get the `UmbracoTestServerTestBase` tests working locally? 🤔 
When trying to follow your steps, I get errors about the `.deps.json` file not existing :(

File: Implementation/Integration-Testing/index.md [hunk 1]
```diff
@@ -143,3 +191,36 @@ public void Ensure_No_Content_After_Doctype_Is_Deleted()
...
+
+```csharp
+[TestFixture]
+public class BackOfficeAssetsControllerTests: UmbracoTestServerTestBase
+{
+    [Test]
+    public async Task EnsureSuccessStatusCode()
+    {
+        // Arrange
+        var url = PrepareUrl<BackOfficeAssetsController>(x=>x.GetSupportedLocales());
```
Comment (Zeegaan) [inline Implementation/Integration-Testing/index.md:207]: We need to use the `PrepareApiControllerUrl` instead 👍 
```suggestion
        var url = PrepareApiControllerUrl<BackOfficeAssetsController>(x => x.GetSupportedLocales());
```

File: Implementation/Integration-Testing/index.md [hunk 2]
```diff
@@ -143,3 +191,36 @@ public void Ensure_No_Content_After_Doctype_Is_Deleted()
...
+}
+```
+
+In this example you have to note two things:
+
+- The PrepareUrl to get the URL of an Action and ensure all services use the URL information when requested. 
+- The Client which is a normal HttpClient, but the base URL points to the test server that is set up for each test.
+
+Note that you can still use GetRequiredService to get the services required to seed data.
+
```
Comment (Zeegaan) [inline Implementation/Integration-Testing/index.md:224]:
```suggestion
In this example you have to note three things:

- You still need the `CustomGlobalSetupTeardown` class
- The PrepareUrl to get the URL of an Action and ensure all services use the URL information when requested. 
- The Client which is a normal HttpClient, but the base URL points to the test server that is set up for each test.

Note that you can still use GetRequiredService to get the services required to seed data.

```

**Review by Zeegaan — CHANGES_REQUESTED** [review]
Got it to work, had a few comments 👍

**Comment by sofietoft** [comment]
Hi all 👋 

I took the liberty to add @Zeegaan s suggestions to this PR.

I'll have to close it though, as it's possible to merge this directly with the docs since the migration to the new platform. I've migrated the PR: https://github.com/umbraco/UmbracoDocs/pull/4721
I apologize for the inconvenience @Mantus667 !
I promise that your PR will still count towards the Umbraco Hacktoberfest challenge 🙌 

Thanks again for helping to update this article!

**Pull request closed without merging**